voice-interface/src/main/java/com/derpgroup/derpwizard/voice/alexa/AlexaRequestType.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (C) 2015 David Phillips
+ * Copyright (C) 2015 Eric Olson
+ * Copyright (C) 2015 Rusty Gerard
+ * Copyright (C) 2015 Paul Winters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.derpgroup.derpwizard.voice.alexa;
+
+/**
+ * Enumeration of the various types of Alexa Requests
+ *
+ * @author Eric
+ * @since 0.0.1
+ */
+public enum AlexaRequestType{
+  LaunchRequest,
+  IntentRequest,
+  SessionEndedRequest;
```

Comment:
Enums are classes and should follow the conventions for classes. Instances of an enum are constants and should follow the conventions for constants.

```
public enum AlexaRequestType {
  LAUNCH_REQUEST,
  INTENT_REQUEST,
  SESSION_ENDED_REQUEST;
```